src/libs/PaymentUtils.js

```diff
@@ -24,6 +25,24 @@ function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
     return validBankAccount || validDebitCard;
 }
 
+/**
+ * @param {String} accountType
```

Comment:
Not A Blocker -- would be nice to describe what we expect here

Reply:
ok, it would be good to add enum values but I noticed it wasn't added in BasePaymentsPage:
https://github.com/Expensify/App/blob/df59e87f510f3a436faf0a77b5a88b03f90a94f8/src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js#L184

Do you still expect to add like this: ?
https://github.com/Expensify/App/blob/df59e87f510f3a436faf0a77b5a88b03f90a94f8/src/styles/StyleUtils.js#L339

Reply:
>Do you still expect to add like this: ?

Yeah or referring to the constants